### PR TITLE
Add to_column and add helper for splitting to num words

### PR DESCRIPTION
**Pull request by eakmanrq**
I removed the doc string tests for to_table and to_column since they didn't really work before due to a recursion issue.

### CODE REVIEW

File: sqlglot/expressions.py [hunk 1]
```diff
@@ -3237,13 +3234,28 @@ def to_table(sql_path, **kwargs):
...
 
 
+def to_column(sql_path: str, **kwargs) -> Column:
+    """
+    Create a column from a `[schema].[table]` sql path. Schema is optional.
+
+    If a column is passed in then that column is returned.
+
+    Args:
+        sql_path: `[schema].[table]` string
```
Comment (tobymao) [inline sqlglot/expressions.py:3249]: it should be [table].[column]

File: sqlglot/helper.py [hunk 2]
```diff
@@ -170,3 +171,27 @@ def find_new_name(taken, base):
...
+        [None, 'db', 'table']
+        >>> split_num_words("db.table", ".", 3, fill_from_start=False)
+        ['db', 'table', None]
+        >>> split_num_words("db.table", ".", 1)
+        ['db', 'table']
+    """
+    words = value.split(sep)
+    if fill_from_start:
+        return [x if x is not None else x for x in [None] * (min_num_words - len(words)) + words]
+    return [x if x is not None else x for x in words + [None] * (min_num_words - len(words))]
```
Comment (tobymao) [inline sqlglot/helper.py:197]: can you reverse all the is not to just x is none

File: sqlglot/expressions.py (hunk 1 above)
Comment (tobymao) [reply sqlglot/expressions.py:3249]: this is similar to def column but i guess it takes a path? i wonder if def column and def table could be merged with to_column and to_table so we don't need both

File: sqlglot/helper.py (hunk 2 above)
Comment (eakmanrq) [reply sqlglot/helper.py:197]: actually that isn't even needed anymore so I will simplify

**Review by tobymao — APPROVED** [review]

**Pull request merged**